app/controllers/internal/broadcasts_controller.rb

```diff
@@ -27,7 +27,7 @@ def index
   private
 
   def broadcast_params
-    params.permit(:title, :processed_html, :type_of, :sent)
+    params.permit(:title, :processed_html, :type_of, :active)
```

Comment:
@benhalpern is this an allowed param in Fastly? It seems super general so I am assuming it is but worth a double check.

Reply:
Huh I don’t think I have fastly access (or at least I’ve never logged in so idk?) but do we usually have to add params into fastly? I don’t have context on how that works, so doing anything with fastly literally never would’ve crossed my mind 🤷🏽‍♀️

Reply:
Will wait to merge/deploy until I hear from @benhalpern

Reply:
`/internal` is exempt from our Fastly param check so we're good to go.